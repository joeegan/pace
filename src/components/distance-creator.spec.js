/* eslint-disable no-unused-vars */

import React from 'react';
import renderer from 'react-test-renderer';
import DistanceCreator from './distance-creator';

it('DistanceCreator changes the text after click', () => {
  const component = renderer.create(
    <DistanceCreator
      handleAdd={() => jest.fn()}
      distanceMap={{foo: 1}}
    />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
