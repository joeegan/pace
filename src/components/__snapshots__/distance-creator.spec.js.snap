exports[`test DistanceCreator changes the text after click 1`] = `
<form
  className={undefined}
  onSubmit={[Function bound handleSubmit]}>
  <input
    autoFocus="true"
    onChange={[Function bound handleDistanceType]}
    value="" />
  <label>
    <input
      checked={true}
      onChange={[Function bound handleRadioChange]}
      type="radio"
      value="k" />
    k
  </label>
  <label>
    <input
      checked={false}
      onChange={[Function bound handleRadioChange]}
      type="radio"
      value="miles" />
    miles
  </label>
  <p
    className="distance-adder_hint">
    Press Enter to submit new distance
  </p>
</form>
`;

exports[`test DistanceCreator changes the text after click 2`] = `
<form
  className={undefined}
  onSubmit={[Function bound handleSubmit]}>
  <input
    autoFocus="true"
    onChange={[Function bound handleDistanceType]}
    value="" />
  <label>
    <input
      checked={true}
      onChange={[Function bound handleRadioChange]}
      type="radio"
      value="k" />
    k
  </label>
  <label>
    <input
      checked={false}
      onChange={[Function bound handleRadioChange]}
      type="radio"
      value="miles" />
    miles
  </label>
  <p
    className="distance-adder_hint">
    Press Enter to submit new distance
  </p>
</form>
`;
